docs/modeling/required-optional.rst

```diff
@@ -0,0 +1,39 @@
+.. include:: /rc1-notice.txt
+
+Required/optional properties
+============================
+
+A property is considered optional if it is valid for it to contain ``null``. If ``null`` is not a valid value to be assigned to a property then it is considered to be a required property.
+
+.. contents:: In this article:
+    :depth: 3
+
+Conventions
+-----------
+
+By convention, a property whose CLR type can contain null will be configured as optional (``string``, ``int?``, ``byte[]``, etc.). Properties whose CLR type can not contain null will be configured as required (``int``, ``decimal``, ``bool``, etc.).
+
+.. note::
+    A property whose CLR type can not contain null can not be configured as optional, it will always be considered required by Entity Framework.
```

Comment:
Two wording nits: 
Cannot should be one word.
The first clause is a complete sentence. Use a semicolon or period.